src/components/list/list.scss

```diff
@@ -37,6 +37,20 @@ $list-item-dense-header-font-size: 1.3rem !default;
 $list-item-dense-font-size: 1.2rem !default;
 $list-item-dense-line-height: 1.25rem !default;
 
+// This is a mixin, which fixes IE11's vertical alignment issue, when using `min-height`.
+// This mixin isn't that much generic to be used in other cases.
+// It can be only used for the list-item's button container.
+// See https://connect.microsoft.com/IE/feedback/details/816293/
+@mixin ie11-min-height-flexbug($min-height) {
+  div.md-button:first-child {
+    &::before {
+      content: '';
+      min-height: $min-height;
+      visibility: hidden;
```

Comment:
Oh, I don't think that's a good idea. It should be located in the list component, since the component can be exported as a (Closure)-/ Module.

In addition to that, it will require much modifying, whenever we change a height of the list. 
Also it's not really an IE fix, it's rather a workaround.